packages/k-ramel/src/createStore/createStore.js

```diff
@@ -38,7 +38,10 @@ export default (definition, options = defaultOptions) => {
       // bind reducer to store definition
       if (driver.getReducer) {
         const { reducer, path } = driver.getReducer() // eslint-disable-line no-unused-vars
-        eval(`definitionWithDrivers${path.length > 0 ? '.' : ''}${path}=reducer`) // eslint-disable-line no-eval
+        if (path.length) {
```

Comment:
Here you will override others definitions key from others drivers I think.

If you have 

createStore({
  ui: { title: types.string() }
})

and then a driver that attemp to add a reducer in "ui.myReducer", you will remove "ui.title" right?

Reply:
Yes, you're right, I forgot that :p